Pointwise.lua

```diff
@@ -69,5 +88,7 @@ end
 
 function Pointwise:clearState()
    self:clearDesc()
+   self._gradOutput = nil
+   nn.utils.clear(self, 'activDesc')
```

Comment:
`activDesc` is niled in `clearDesc` already, should be `_gradOutput` there instead so that `nn.utils.clear` can call `set` on it